google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -62,6 +62,9 @@ def insert_violations(self, violations, resource_name,
             # Create the violations snapshot table.
             snapshot_table = self._create_snapshot_table(
                 resource_name, snapshot_timestamp)
+        except MySQLdb.OperationalError, e:
+            LOGGER.warning('Violations table already exists: %s', e)
+            snapshot_table = resource_name + '_' + snapshot_timestamp
```

Comment:
Wouldn't it be easier to use `_get_snapshot_table()` here? Since the method already exists.